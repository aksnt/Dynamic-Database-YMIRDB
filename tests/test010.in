SET a 1 2 3
SET b 4 5 a
SET c a b
FORWARD c
BACKWARD b
MAX c
MIN c
LEN c
SUM c
TYPE c
BYE
