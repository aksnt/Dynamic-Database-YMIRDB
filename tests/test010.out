> ok

> ok

> ok

> a, b

> c

> 5

> 1

> 8

> 21

> general

> bye
